build/bazelutil/BUILD.bazel

```diff
@@ -10,6 +10,19 @@ genrule(
     stamp = True,
 )
 
+NOGO_FAILED_TEST_OUTPUT = "echo 'must use exactly one of `--config nogo` or `--config nonogo` explicitly'; exit 1"
+
+genrule(
+    name = "test_nogo_configured",
+    outs = ["empty.txt"],
+    cmd = select({
```

Comment:
Rather than running the genrule with echo and exit 1, you can make the build fail here by removing the both_nogo_and_nonogo and conditions:default branches, then putting the error message in select's no_match_error. That makes it a bit easier to see what's going on here!